packages/xrpl/test/models/DIDDelete.test.ts

```diff
@@ -0,0 +1,28 @@
+import { assert } from 'chai'
+
+import { validate } from '../../src'
+import { validateDIDDelete } from '../../src/models/transactions/DIDDelete'
+
+/**
+ * DIDDelete Transaction Verification Testing.
+ *
+ * Providing runtime verification testing for each specific transaction type.
+ */
+describe('DIDDelete', function () {
+  let tx
+
+  beforeEach(function () {
+    tx = {
+      Account: 'rfmDuhDyLGgx94qiwf3YF8BUV5j6KSvE8',
+      Fee: '10',
+      Flags: 2147483648,
+      Sequence: 4,
+      TransactionType: 'DIDDelete',
+    } as any
```

Comment:
Why is typed as `any` instead of `DIDDelete`?

Reply:
So that you can remove required fields without TS complaining (see other model tests)